idea_town/frontend/static-src/app/views/details-view.js

```diff
@@ -0,0 +1,83 @@
+import Model from 'ampersand-model';
+import Collection from 'ampersand-collection';
+import CollectionRenderer from 'ampersand-collection-view';
```

Comment:
If you use [`renderCollection()`](https://github.com/AmpersandJS/ampersand-view/blob/master/ampersand-view.js#L363) you don't need to import `ampersand-collection-view` here like this to create a generic renderer